Lib/test/test_pstats.py

```diff
@@ -30,12 +34,40 @@ class StatsTestCase(unittest.TestCase):
     def setUp(self):
         stats_file = support.findfile('pstats.pck')
         self.stats = pstats.Stats(stats_file)
+        self.temp_storage = tempfile.NamedTemporaryFile(delete=False)
+        script_string = 'import os'
+        cProfile.run(script_string, filename=self.temp_storage.name)
```

Comment:
I don't think there's need to have the variable `script_string` either. `cProfile.run('import os', filename=XX)` is more than clear to me.

Reply:
Got it